src/signalrclient/hub_connection_impl.cpp

```diff
@@ -357,6 +357,12 @@ namespace signalr
 
             for (const auto& val : messages)
             {
+                // Protocol received an unknown message type and gave us a null object, safe to ignore
+                if (val == nullptr)
+                {
+                    continue;
```

Comment:
It'd be nice to just not do this. If we think other protocols might include null messages in the vector, maybe we should log a warning so we can get rid of this later.

Reply:
In .NET we throw an exception which closes the connection (server or client side, same behavior). We could throw here as well which will close the connection to align behavior.